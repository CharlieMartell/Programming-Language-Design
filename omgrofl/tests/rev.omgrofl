loooooooool iz 69
looooool iz 32
rofl loooooooool
lool iz 110
rofl lool
lmao lool 
lmao lool 
lmao lool
lmao lool 
loool iz lool 
lmao lool 
lmao lool 
rofl lool
rofl 101
rofl loool
rofl looooool
rofl 65 
rofl 32 
rofl 78
rofl 117 
rofl 109
rofl 98
rofl 101
rofl 114
rofl 32
rofl 84
rofl 111
rofl 32
rofl 82
rofl 101
rofl 118
rofl 101
rofl 114
rofl 115
rofl 101
rofl 32
stfw lol
loooool iz lol
rtfm
lool iz loooool
loooool to /dev/null
rtfm
loool iz 10
looool to /dev/null
rtfm
wtf lool iz liek 0
tldr
brb
wtf loool iz liek 0
tldr
brb
lulz loool
lulz lool
lmao looool
brb
wtf looool iz liek 10
looool to /dev/null
lmao loooool
brb
wtf lool iz liek 0
tldr
brb
brb
lol iz 48
rtfm
wtf lol iz liek 0
tldr
brb
lmao looool
lulz lol
brb
rofl looool
wtf loooool iz liek 0
tldr
brb
brb