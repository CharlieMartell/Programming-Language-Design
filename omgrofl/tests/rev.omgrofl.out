Enter A Number To Reverse 422
